content/kubermatic/master/concepts/roles/_index.en.md

```diff
@@ -0,0 +1,37 @@
++++
+title = "Predefined Roles"
+date = 2020-04-02T12:07:15+02:00
+weight = 10
+pre = "<b></b>"
++++
+
+## Overview
+
+A project is an entity that holds various resources.  All resources in a project are equal in terms of the `groups` attached
+to them. Affiliation of a `user` to one of the `groups` give them certain powers they are allowed to use within a project.
+
+### Kubermatic roles
+
+There are three roles: owner, editor, and viewer. These roles are concentric; that is, the owner role includes the permissions
+in the editor role, and the editor role includes the permissions in the viewer role.
+
+  - **viewer**: can only read
+  
+  - **editor**: viewer + can create, edit & delete cluster
+  
+  - **owner**: editor + manage permissions for a project
```

Comment:
Owner can do everything that the editor can do and additionally manage permissions for the project.